var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -44,12 +44,16 @@ class Metis(Package):
     version('5.0.2', 'acb521a4e8c2e6dd559a7f9abd0468c5')
     version('4.0.3', 'd3848b454532ef18dc83e4fb160d1e10')
 
-    variant('shared', default=True, description='Enables the build of shared libraries.')
-    variant('debug', default=False, description='Builds the library in debug mode.')
+    variant('shared', default=True,
+            description='Enables the build of shared libraries.')
+    variant('debug', default=False,
+            description='Builds the library in debug mode.')
     variant('gdb', default=False, description='Enables gdb support.')
 
-    variant('int64', default=False, description='Sets the bit width of METIS\'s index type to 64.')
-    variant('real64', default=False, description='Sets the bit width of METIS\'s real type to 64.')
+    variant('int64', default=False,
+            description='Sets the bit width of METIS\'s index type to 64.')
+    variant('real64', default=False,
+            description='Sets the bit width of METIS\'s real type to 64.')
```

Comment:
For the record, we exempt things like URLs and `variant()` directives from the line-length checks, so these don't have to be under 80 characters.